source/_docs/automation/condition.markdown

```diff
@@ -3,7 +3,7 @@ title: "Automation Conditions"
 description: "Automations can test conditions when invoked."
 ---
 
-Conditions are an optional part of an automation rule and can be used to prevent an action from happening when triggered. When a condition does not return true, the automation will stop executing. Conditions look very similar to triggers but are very different. A trigger will look at events happening in the system while a condition only looks at how the system looks right now. A trigger can observe that a switch is being turned on. A condition can only see if a switch is currently on or off.
+Conditions are an optional part of an automation rule. They can be used to prevent the automation's actions from being run. After an automation has been triggered, all conditions will be checked. If any of them do not return true, the automation will stop executing. Conditions look very similar to triggers, but they are very different &mdash; a trigger will look at events happening in the system, while a condition only looks at how the system looks right now. A trigger can observe that a switch is being turned on. A condition can only see if a switch is currently on or off.
```

Comment:
This wording makes it more confusing. 

> After an automation has been triggered

Automation is only triggered if the conditions pass. This wording makes the last triggered time look wrong as well.

../Frenck

/CC @c0ffeeca7

Reply:
Good point. Let me fix this.